falcon/asgi/structures.py

```diff
@@ -111,7 +112,17 @@ def __init__(
 
         self.comment = comment
 
-    def serialize(self):
+    def serialize(self, handler=None):
+        """Serialize this event to string.
+
+        Args:
+            handler: Handler object that will be used to serialize the ``json`` attribute to
+                string. When not provided a default handler using the python builtin
```

Comment:
Capitalize Python, or just remove it altogether IMHO. The reader will know which language we are targeting here.